### PR TITLE
Packaging from library pom

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: library/pom.xml [hunk 1]
```diff
@@ -1026,5 +1021,292 @@
...
+                            <attach>false</attach>
+                            <descriptorRefs>
+                                <descriptorRef>distribution</descriptorRef>
+                            </descriptorRefs>
+                        </configuration>
+                        <dependencies>
+                            <dependency>
+                                <groupId>com.proofpoint.platform</groupId>
+                                <artifactId>packaging</artifactId>
+                                <version>1.34-SNAPSHOT</version>
```
Comment (glent) [inline library/pom.xml]: Should this be project.version?

Comment (johngmyers) [inline library/pom.xml]: Wouldn't that pull in the version of the client project, not platform?

The release plugin appears to update this value.

**Comment by glent** [comment]
LGTM

**Comment by johngmyers** [comment]
Rebased and merged

**Pull request closed without merging**